tests/doc-comments-help.rs

```diff
@@ -108,3 +108,50 @@ fn splits_subcommand_doc_comment_between_short_and_long() {
     assert!(long_help.contains("DO NOT PASS A BAR UNDER ANY CIRCUMSTANCES"));
     assert!(long_help.contains("Or something else"));
 }
+
+#[test]
+fn no_mangle_doc() {
+    /// TOUGH!
+    ///
+    /// ---------------------------
+    /// !            -            !
+    /// !           { }           !
+    /// !           | |           !
+    /// !           | |           !
+    /// !        .-.! !.-.        !
+    /// !      .-!  ! !  !.-.     !
+    /// !      ! !       !  ;     !
+    /// !      \           ;      !
+    /// !       \         ;       !
+    /// !        !       :        !
+    /// !        !       |        !
+    /// !        |       |        !
+    /// !                         !
+    /// ---------------------------
```

Comment:
Even if it's quite a great ASCII art, I'd prefer a more neutral draw.

Reply:
Nobody respects art these days...